modules/mobile/app/helpers/mobile/list_filter.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
```

Comment:
You may notice that the inputs are not being validated. That will come in the next ticket.